### PR TITLE
Format edit profile

### LINKED ISSUE #39: Auto Populate Form
When the user accesses the `form` to change their information, the form should appear with the user's current information in the form already. 

_Reasoning: Upon submit, all information will be replaced._

### PR DESCRIPTION
**Pull request by kathrynljackson**
### Participating Group Members:
* KJ
 
### Code Highlights:
 * Format Button and Input, changed tags to match
* Change submit button type on the `Edit Profile` to "submit"
* Rendered user info on `Profile` is now dynamic
* Existing `user` info is now used to autopopulate the `EditProfile` component. According to the Redux Tools, changes made in the form through the interface are reflected in the state. 
* If that value does not exist, the form shows the placeholder
* Move EditProfile Components, add annotations
* Create toggle function- unfinished

### Have Tests Been Added?
- [x] No.
- [ ] Yes, but all tests are not passing. 
- [ ] Yes, and all are passing.
 
### Any background context you want to provide?
* I changed the some of the tags so that the format/styling would be uniform with existing code across the app.
 
### Message/Questions for reviewer:
* Do you notice any styling changes that affect the new functionality changes?
 
### Issues: 
* Addresses #20 #29 #18 
* Closes #39 
 
### Screenshots (if appropriate): 
Screen Shot 2020-12-12 at 2 01 09 PM

 
### Tracking Consistency:
- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

File: code/web/src/modules/user/Profile.js [hunk 1]
```diff
@@ -42,32 +42,58 @@ const Profile = (props) => (
       </GridCell>
       <GridCell style={{ padding: '2em', textAlign: 'center' }}>
         <H4 style={{ marginBottom: '0.5em' }}>{props.user.details.name}</H4>
-        <p style={{ marginBottom: '0.5em' }}>Personal description goes here</p>
-        {/*Description text should be dynamic from user details ie props.user.details.description */}
+        <p style={{ marginBottom: '0.5em' }}>{props.user.details.description || 'I\'m a trendy Crate user!'}</p>
```
Comment (aemcdonald) [inline code/web/src/modules/user/Profile.js:45]: Great sad path handling here if a description does not exist

File: code/web/src/modules/user/EditProfile.js [hunk 2]
```diff
@@ -8,51 +14,59 @@ class EditProfile extends Component {
...
             //image: ?
-            name: '',
-            description: '',
-            email: '',
-            address: ''
+            name: props.user.details.name || '',
+            description: props.user.details.description || '',
+            email: props.user.details.email || '',
+            address: props.user.details.address || ''
         }
```
Comment (stacyp2006) [inline code/web/src/modules/user/EditProfile.js:21]: Nice work on this. I like that the user will be able to choose what to edit without having to edit the entire user profile.

**Review by stacyp2006 — APPROVED** [review]
Form looks great, KJ.

**Review by aemcdonald — APPROVED** [review]

**Review by nathanielmillard — APPROVED** [review]
This look great!
I know we have a little overlap, but I think it should be easy to resolve :)

**Comment by TravisGM92** [comment]
There are some conflicting files here and since I wasn't working on it, I won't resolve them. Once those are resolved I'd be happy to merge

File: code/web/src/modules/user/EditProfile.js [hunk 3]
```diff
@@ -8,51 +14,59 @@ class EditProfile extends Component {
         super()
         this.state = {
             //image: ?
```
Comment (TravisGM92) [inline code/web/src/modules/user/EditProfile.js:16]: Just added this attribute to the User model. Once we figure out the bug in my PR, image will be available to edit. It'll come with a default image URL as well

**Comment by kathrynljackson** [comment]
> There are some conflicting files here and since I wasn't working on it, I won't resolve them. Once those are resolved I'd be happy to merge

All merge conflicts have been resolved.

**Comment by nathanielmillard** [comment]
> There are some conflicting files here and since I wasn't working on it, I won't resolve them. Once those are resolved I'd be happy to merge

Conflicts should be handled ✔️

**Pull request merged**